src/library/scala/util/Either.scala

```diff
@@ -245,53 +234,49 @@ sealed abstract class Either[+A, +B] extends Product with Serializable {
     case Left(_)  => false
   }
 
-  /**
-   * Returns `true` if `Left` or returns the result of the application of
-   * the given predicate to the `Right` value.
+  /** Returns `true` if `Left` or returns the result of the application of
+   *  the given predicate to the `Right` value.
    *
-   * {{{
-   * Right(12).forall(_ > 10) // true
-   * Right(7).forall(_ > 10)  // false
-   * Left(12).forall(_ > 10)  // true
-   * }}}
+   *  {{{
+   *  Right(12).forall(_ > 10) // true
+   *  Right(7).forall(_ > 10)  // false
+   *  Left(12).forall((_: Int) > 10) // true
```

Comment:
this is a tricky one — it's true because it's a `Left`, not because 12 is greater than 10. `Left(12).forall((_: Int) < 10)` is true, too. maybe this could be made more explicit?